app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/Data/AdminEnhancedMediaGalleryCategoryData.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="urn:magento:mftf:DataGenerator/etc/dataProfileSchema.xsd">
+    <entity name="TestSubCategory" type="category">
```

Comment:
What is the reason for introducing this entity instead of `SimpleSubCategory`?

Reply:
@sivaschenko there were instances when the test would fail because the category name did not match, to be more specific, it was the hashed suffix on the category name that did not match so I introduced the new entity.

Reply:
@joweecaquicla what was the reason of not matching hashed suffix?

Reply:
@sivaschenko I was not able to investigate that issue thoroughly as I was not able to replicate it in my local environment.